### PR TITLE
Better docs of returned tensor in ctc_ops.py

### PR DESCRIPTION
**Pull request by igormq**
'Logits' aren't a meaningful word for what the ctc_cost function returns. Browsing the implementation (tensorflow/tensorflow/core/util/ctc/) I saw that the cost function is returning the negative log probabilities of the target labelling, so, this new comment erases any doubt. Thanks!

### CODE REVIEW

**Comment by igormq** [comment]
@googlebot I signed it!

File: tensorflow/python/ops/ctc_ops.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def ctc_loss(inputs, labels, sequence_length,
     ctc_merge_repeated: Boolean.  Default: True.
 
   Returns:
-    A 1-D `float` `Tensor`, size `[batch]`, containing logits.
+    A 1-D `float` `Tensor`, size `[batch]`, containing the minus log probabilities.
```
Comment (rmlarsen) [inline tensorflow/python/ops/ctc_ops.py]: I would write "the negative log probabilities".

Comment (igormq) [inline tensorflow/python/ops/ctc_ops.py]: I agree! :)

**Comment by rmlarsen** [comment]
@ebrevdo

**Comment by rmlarsen** [comment]
Oops, hit close by mistake.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please
